packages/xpetra/src/CrsMatrix/Xpetra_TpetraCrsMatrix_decl.hpp

```diff
@@ -1249,7 +1249,7 @@ namespace Xpetra {
 
     /// \brief Access the local Kokkos::CrsMatrix data
     typename local_matrix_type::HostMirror getLocalMatrixHost () const {
-      TEUCHOS_UNREACHABLE_RETURN(local_matrix_type::HostMirror());
+      TEUCHOS_UNREACHABLE_RETURN(typename local_matrix_type::HostMirror());
```

Comment:
Why didn't this get caught by existing builds?

Reply:
The only reason I can think of is that I built with a newer GCC than before.